core/src/components/input/input.ionic.scss

```
@@ -54,3 +54,39 @@
     --border-color: #{tokens.$ionic-color-neutral-600};
   }
 }
+
+// Input - Disabled
+// ----------------------------------------------------------------
+
+:host(.input-disabled) {
+  // color for the text within the input
+  --color: #{tokens.$ionic-color-neutral-400};
+  --background: #{tokens.$ionic-color-neutral-10};
```

```suggestion
  --background: #{tokens.$ionic-color-neutral-10};
  
  pointer-events: none;
```

This needs to be added to prevent the hover state on disabled inputs